docs/install_guides/installing-openvino-apt.md

```diff
@@ -1,10 +1,216 @@
+
+## System Requirements
+
+@sphinxdirective
+.. tab:: Operating Systems
+
+  * Ubuntu 18.04 long-term support (LTS) x86, 64-bit
+  * Ubuntu 20.04 long-term support (LTS) x86, 64-bit
+
+.. tab:: Hardware
+
+  Optimized for these processors:
+
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel Atom® processor with support for Intel® Streaming SIMD Extensions 4.1 (Intel® SSE4.1)
+  * Intel Pentium® processor N4200/5, N3350/5, or N3450/5 with Intel® HD Graphics
+  * Intel® Iris® Xe MAX Graphics
+  * Intel® Neural Compute Stick 2
+
+.. tab:: Processor Notes
+
+  Processor graphics are not included in all processors.
+  See `Product Specifications`_ for information about your processor.
+
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
+
+@endsphinxdirective
+
+## Installing OpenVINO Runtime
+
+### Step 1: Set Up the OpenVINO Toolkit APT Repository
+
+1. Install the GPG key for the repository
+
+    a. Download the [GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB](https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB).
+
+        You can also use the following command:
+        ```sh
+        wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+        ```
+
+    b. Add this key to the system keyring:
+        ```sh
+        sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+        ```
+
+    > **NOTE**: You might need to install GnuPG: `sudo apt-get install gnupg`
+
+2.	Add the repository via the following command:
+@sphinxdirective
+
+.. tab:: Ubuntu 18
+
+   .. code-block:: sh
+
+      echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+
+.. tab:: Ubuntu 20
+
+   .. code-block:: sh
+
+      echo "deb https://apt.repos.intel.com/openvino/2022 focal main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+
+@endsphinxdirective
+
+
+3.	Update the list of packages via the update command:
+   ```sh
+   sudo apt update
+   ```
+
+4.	Verify that the APT repository is properly set up. Run the apt-cache command to see a list of all available OpenVINO packages and components:
+   ```sh
+   apt-cache search openvino
+   ```
+
+
+### Step 2: Install OpenVINO Runtime Using the APT Package Manager
+
+#### Install OpenVINO Runtime
+
+@sphinxdirective
+
```

Comment:
Let's also note, that users can use `–no-install-recommends` option to install only required packages, but in this case samples cannot be compiled because build tools are not installed.

Reply:
Have applied a note regarding this option with https://github.com/openvinotoolkit/openvino/pull/14492/commits/fd16a27d39aff668654e462b42cb8601909aad9b